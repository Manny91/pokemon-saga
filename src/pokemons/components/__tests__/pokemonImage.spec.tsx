import React from "react";
import { PokemonImage } from "../pokemonImage";
import "@testing-library/jest-dom/extend-expect";
import { cleanup, render } from "@testing-library/react";
import "jest-styled-components";

describe("PokemonImage Component", () => {
  afterEach(cleanup);
  it("should render properly", () => {
    const subject = render(<PokemonImage id={"1"} />);
    const tree = subject.container;

    expect(tree).toMatchSnapshot();
  });
});
