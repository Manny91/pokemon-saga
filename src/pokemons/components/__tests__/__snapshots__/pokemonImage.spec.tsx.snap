// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PokemonImage Component should render properly 1`] = `
.c0 {
  image-rendering: pixelated;
}

<div>
  <img
    class="c0"
    src="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/1.png"
  />
</div>
`;
